quickwit/quickwit-actors/src/mailbox.rs

```diff
@@ -523,9 +529,9 @@ mod tests {
             )
             .await
             .unwrap();
-        assert_eq!(backpressure_micros_counter.get(), 0u64);
+        assert!(backpressure_micros_counter.get() < 500);
```

Comment:
With this fix, we can't get without adding a bit of latency because of the additional "task" switch. Measured at less than 100 µs on my laptop (debug mode).